src/main/scala/metagenomica/loquats/3.blast.scala

```diff
@@ -23,11 +23,20 @@ case class blastDataProcessing[MD <: AnyMG7Parameters](val md: MD)
 extends DataProcessingBundle(
   bundles.blast,
   md.referenceDB
-)(
-  input = data.blastInput,
+)(input  = data.blastInput,
   output = data.blastOutput
 ) {
 
+  def filterResult(blastResult: File): Iterator[Seq[String]] = {
+    val csvReader = csv.Reader(md.blastOutRec.keys, blastResult)
+
+    val filtered = csvReader.rows.filter(md.blastFilter)
+
+    csvReader.close()
```

Comment:
I'm not sure I understand that. But I can rearrange code so that this will be closed in the end of an iteration.